corehq/apps/domain/models.py

```diff
@@ -616,6 +618,7 @@ def case_sharing_included(self):
         return self.case_sharing or reduce(lambda x, y: x or y, [getattr(app, 'case_sharing', False) for app in self.applications()], False)
 
     def save(self, **params):
+        self.last_modified = datetime.utcnow()
```

Comment:
@nickpell this is correct usage according to the conventions I laid out and am trying to enforce. Specifically, it's using an offset-naive datetime representing UTC to represent "server time" / "absolute time". The `ServerTime` etc. utilities are for converting between conceptual types of time, i.e. localizing a server time.